pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -151,6 +151,7 @@ type EC2Metadata interface {
 type VolumeOptions struct {
 	CapacityGB int
 	Tags       *map[string]string
+	VolumeType string
```

Comment:
This, I think, insufficiently expresses what EBS offers.  There is also Encryption and IOPS.

The other 2 will have their own attributes that might not be supported by the others.  This is why a generic cloud API might not make sense.  Each might have to be its own in order to fully access all features of that provider.